src/app-layer.c

```diff
@@ -99,6 +101,18 @@ static void AppLayerIncFlowCounter(uint8_t ipproto, AppProto alproto, ThreadVars
     }
 }
 
+void AppLayerIncTxCounter(uint8_t ipproto, AppProto alproto, ThreadVars *tv,
+                          uint64_t step)
+{
+    if (!AppLayerParserProtocolIsTxAware(ipproto, alproto))
```

Comment:
if we already check this prior to calling this function, then we can probably leave this check out here